src/gameconfigfile.cpp

```diff
@@ -156,14 +156,14 @@ FGameConfigFile::FGameConfigFile ()
 	{
 		SetSection("SoundfontSearch.Directories", true);
```

Comment:
Not entirely sure but I think a change here meant that the included gzdoom.sf2 soundfont will no longer load from the build/soundfonts library and only paths specified in the ini file were valid. I lost midi sound until I copied the gzdoom.sf2  to ~/.config/gzdoom/soundfonts lib which was a valid path in the ini file. This on a linux system. Was that intentional?

Reply:
well, that line wasn't changed, all i added in that file was GetChars to the places where it wasn't compiling